### PR TITLE
Updated to v10

### PR DESCRIPTION
**Pull request by Chris3606**
The purpose of this PR is to do a fairly minimal set of changes that update the integration library to SadConsole v10.

### CODE REVIEW

**Review by Thraka — CHANGES_REQUESTED** [review]
Few things to note and wait on the next update to SC

File: TheSadRogue.Integration/Maps/MapTerrainCellSurface.cs [hunk 1]
```diff
@@ -29,7 +29,23 @@ public class MapTerrainCellSurface : GridViewBase<ColoredGlyph>, ICellSurface
...
+        {
+            get
+            {
+                var array = new ColoredGlyphBase[range.End.Value - range.Start.Value];
+                for (int i = 0; i < array.Length; i++)
+                    array[i] = this[range.Start.Value + i];
+
+                return array;
+            }
+        }
```
[the remark this answers is not in the document]
Comment (Chris3606) [reply TheSadRogue.Integration/Maps/MapTerrainCellSurface.cs]: Removed

File: ExampleGame/Program.cs [hunk 2]
```diff
@@ -52,6 +61,7 @@ private static void Init()
             Map.IsFocused = true;
 
             // Destroy the default starting console that SadConsole created automatically because we're not using it.
+            // TODO: May not be necessary in v10
             GameHost.Instance.DestroyDefaultStartingConsole();
```
[the remark this answers is not in the document]
Comment (Chris3606) [reply ExampleGame/Program.cs]: Removed

**Comment by Thraka** [comment]
I just pushed an update to the branch. Also, the template is pointing to the local copy of the integration library. Since I didn't know what version you wanted to use, so you'll need to modify the project files with new versions and remove those local references.

Changes:

1. I updated the individual templates to match the default SadConsole template.
2. Changed a few comments where there were double spaces to single spaces.
3. Startup modernized and cleaned up.
4. MonoGame template has the app manifest file.
5. Because of implicit usings, removed SadConsole namespaces where appropriate.

**Review by Thraka — APPROVED** [review]

**Pull request merged**